src/components/profile/ProfileOtherAdminCard.tsx

```diff
@@ -0,0 +1,246 @@
+import { useMutation, useQuery } from '@apollo/react-hooks'
+import { Button } from '@chakra-ui/react'
+import { Form, message, Typography } from 'antd'
+import { CardProps } from 'antd/lib/card'
+import { FormComponentProps } from 'antd/lib/form'
+import gql from 'graphql-tag'
+import { useAuth } from 'lodestar-app-element/src/contexts/AuthContext'
+import React, { FormEvent, useState } from 'react'
+import { defineMessages, useIntl } from 'react-intl'
+import hasura from '../../hasura'
+import { handleError } from '../../helpers'
+import { commonMessages } from '../../helpers/translation'
+import AdminCard from '../common/AdminCard'
+import MigrationInput from '../common/MigrationInput'
+import { StyledForm } from '../layout'
+const messages = defineMessages({
+  otherInfoTitle: { id: 'member.messages.ui.otherInfoTitle', defaultMessage: '其他資料' },
+  phone: { id: 'member.messages.ui.phone', defaultMessage: '手機號碼' },
+  enterPhone: { id: 'member.messages.ui.enterPhone', defaultMessage: '請輸入手機號碼' },
+  enter: { id: 'member.messages.ui.messages.enter', defineMessages: '請輸入' },
+})
+
+const GET_POHNE_ENABLE_SETTING = gql`
+  query GET_POHNE_ENABLE_SETTING {
+    setting(where: { key: { _eq: "profile_page.edit_phone.enable" } }) {
+      key
+      app_settings {
+        id
+        key
+        value
+      }
+    }
+  }
+`
+const GET_MEMBER_PHONE = gql`
+  query GET_MEMBER_PHONE($memberId: String!) {
+    member_phone(where: { member_id: { _eq: $memberId } }) {
+      id
+      member_id
+      phone
+    }
+  }
+`
```

Comment:
NITS: prefer put it at the bottom fo the file

Reply:
change it